container/docker/Dockerfile.base

```diff
@@ -1,5 +1,14 @@
 FROM alpine:latest
 
 RUN apk add openjdk11-jdk
+RUN apk add nodejs
+RUN apk add npm
 RUN apk add curl
+COPY . /usr/src/yavin
 COPY yavin-run.sh /usr/local/bin/
```

Comment:
Do we need this and `curl`?

Reply:
yea i thought in case someone ends up trying the script. curl addition to image is lightweight.